hepcrawl/spiders/arxiv_spider.py

```diff
@@ -9,14 +9,22 @@
 
 """Spider for arXiv."""
 
+import re
+
 from scrapy import Request, Selector
 from scrapy.spiders import XMLFeedSpider
-from ..mappings import CONFERENCE_PAPERS
-from ..utils import get_first
+from ..mappings import CONFERENCE_WORDS
+from ..mappings import THESIS_WORDS
+from ..mappings import LICENSES
+from ..utils import split_fullname
+from ..utils import coll_cleanforthe
 
 from ..items import HEPRecord
 from ..loaders import HEPLoader
 
+RE_CONFERENCE = re.compile(r'\b(%s)\b' % '|'.join(CONFERENCE_WORDS), re.I | re.U)
```

Comment:
Here and the next line, is safer to escape with `re.escape(CONFERENCE_WORDS)`, in order to avoid special characters to be interpreted by the regexp.